app/controllers/dashboards_controller.rb

```diff
@@ -25,7 +25,7 @@ def authenticate_with_api_key
     hashed_key = Digest::SHA256.hexdigest(params_key)
     return unless (@api_key = ApiKey.unexpired.find_by_hashed_key(hashed_key))
 
-    set_tags "gemcutter.user.id" => @api_key.user_id, "gemcutter.user.api_key_id" => @api_key.id
+    set_tags "gemcutter.owner" => @api_key.owner.to_gid, "gemcutter.api_key" => @api_key.to_gid
```

Comment:
Will this change statistics or behavior somewhere or am I misunderstanding the purpose of these tags?

Reply:
the tags are for logs / error tagging / datadog tracing context. purely for observability debugging